### PR TITLE
DOC: Improve documentation in `Common` operator classes

### PR DESCRIPTION
**Pull request by jhlegarreta**
Improve documentation in `Common` operator classes:
- Add documentation to type aliases/methods that are usually documented in
  the same manner.
- Remove documentation from methods not needing it (e.g. the default
  constructor).
- Remove excessive white spaces.
- Fix typos.
- Reduce the documentation block span if the documentation fits into a
  single line.

## PR Checklist
- [X] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [X] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)

### CODE REVIEW

File: Modules/Core/Common/include/itkLaplacianOperator.h [hunk 1]
```diff
@@ -104,16 +101,16 @@ class ITK_TEMPLATE_EXPORT LaplacianOperator : public NeighborhoodOperator<TPixel
   SetDerivativeScalings(const double * s);
 
 protected:
-  /** Typedef support for coefficient vector type.  Necessary to
-   * work around compiler bug on VC++.   */
+  /** Typedef support for coefficient vector type. Necessary to
```
Comment (Leengit) [inline Modules/Core/Common/include/itkLaplacianOperator.h:104]: `Typedef` -> `Alias`

File: Modules/Core/Common/include/itkSobelOperator.h [hunk 2]
```diff
@@ -133,22 +132,16 @@ class ITK_TEMPLATE_EXPORT SobelOperator : public NeighborhoodOperator<TPixel, VD
   }
 
 protected:
-  /**
-   * Typedef support for coefficient vector type.  Necessary to
-   * work around compiler bug on VC++.
-   */
+  /** Typedef support for coefficient vector type. Necessary to
```
Comment (Leengit) [inline Modules/Core/Common/include/itkSobelOperator.h:135]: `Alias`

**Review by Leengit — APPROVED** [review]
Looks good.

**Review by dzenanz — APPROVED** [review]
Looks good. Please apply Lee's suggestions.

File: Modules/Core/Common/include/itkLaplacianOperator.h (hunk 1 above)
Comment (jhlegarreta) [reply Modules/Core/Common/include/itkLaplacianOperator.h]: Changed in b9ee030.

File: Modules/Core/Common/include/itkSobelOperator.h (hunk 2 above)
Comment (jhlegarreta) [reply Modules/Core/Common/include/itkSobelOperator.h]: Changed in b9ee030.

**Comment by jhlegarreta** [comment]
> Looks good. Please apply Lee's suggestions.

Thanks for the reviews. Addressed in b9ee030.

**Pull request merged**